api/src/utils/validate-storage.ts

```diff
@@ -29,8 +30,8 @@ export async function validateStorage(): Promise<void> {
 	}
 
 	try {
-		await access(env['EXTENSIONS_PATH'], constants.R_OK);
+		await access(getExtensionsPath(), constants.R_OK);
 	} catch {
-		logger.warn(`Extensions directory (${path.resolve(env['EXTENSIONS_PATH'])}) is not readable!`);
+		logger.warn(`Extensions directory (${path.resolve(getExtensionsPath())}) is not readable!`);
```

Comment:
When running locally for the first time, this warning will surface as the temp extension folder hasn't been created yet.

Should we add a check here on whether `EXTENSIONS_LOCATION` is set?

Reply:
I agree, something along these lines would be in order i think 🤔 
```suggestion
        if (env['EXTENSIONS_LOCATION'] === 'local') {
		try {
			await access(getExtensionsPath(), constants.R_OK);
		} catch {
			logger.warn(`Extensions directory (${path.resolve(getExtensionsPath())}) is not readable!`);
		}
	}
```

Reply:
Thanks for the heads up, patched via https://github.com/directus/directus/pull/20207/commits/e47066604ba50e0bf6f02a7175808b25dc05fab5.

I think it should be bypassed for all, when `EXTENSIONS_LOCATION` is set, as the storage driver name can be any string including `local`.

The path is validated when the temp extensions folder is created during the sync.
https://github.com/directus/directus/blob/e47066604ba50e0bf6f02a7175808b25dc05fab5/api/src/extensions/lib/sync-extensions.ts#L47-L48